src/Insect/Environment.purs

```diff
@@ -108,8 +108,17 @@ initialEnvironment =
         [ constFunc  "abs" (Q.abs >>> pure)
         , constFunc  "acos" Q.acos
         , constFunc  "acosh" Q.acosh
-        , constFunc  "acos" Q.acos
-        , constFunc  "acosh" Q.acosh
+        , constFunc  "acot" Q.acot
+        , constFunc  "arccotangent" Q.acot
+        , constFunc  "acoth" Q.acoth
+        , constFunc  "arccotangenth" Q.acoth
```

Comment:
I think this "h" at the end of the full names of the hyperbolic functions may be a little awkward. Thoughts?

Reply:
They do look a bit weird, yes. I don't have better suggestions though. Considered options:

* A `hyperbolic` or `hyperbolic_` prefix? Probably too long
* Remove/comment-out the long names for now until we have a better solution? That would be weird in terms of consistency

Reply:
> A hyperbolic or hyperbolic_ prefix? Probably too long

How about a `hyp` or `hyp_` prefix?

Reply:
Okay. Let's go with `hyp`. Thank you